### DISCUSSION TITLE
Convert CAPT v2 to v1?

### DISCUSSION THREAD

**Discussion by traghav** [mounaiban/captdriver#42] [Q&A] [answered]
Hi!
Incredible tool, thanks for all the work you've been doing.
I have some CAPT print streams captured and want to convert it back to raster/png/anything readable
It's unfortunately CAPT v2 with HiSCOA encoding. From what I understand, this hasn't been reverse engineered yet?

Are there any ways I could get the original data? Maybe convert it back to v1 and use your decoder tool?
I tried to change the print processor, but it doesn't output anything other than capt v2.
Any other thing I could try?

Thanks

**Comment by mounaiban** [comment, marked as answer]
Duplicate of https://github.com/mounaiban/studycapt/discussions/2
